File: src/tasks/Tokenizer/Tokenizer.cs
using System;

class Tokenizer
{
    static void Main()
    {
        string str = "Hello,How,Are,You,Today"; 
		// or Regex.Split ( "Hello,How,Are,You,Today", "," );
		// (Regex is in System.Text.RegularExpressions namespace
		string[] strings = str.Split(',');
		foreach (string s in strings)
		{
			System.Console.WriteLine (s + ".");
		}
    }
}

File: src/tasks/Tokenizer/Tokenizer.java
import java.util.StringTokenizer;

public class Tokenizer {
	public static void main(String[] args) {
		String toTokenize = "Hello,How,Are,You,Today";
		StringTokenizer tokenizer = new StringTokenizer(toTokenize, ",");
		
		while(tokenizer.hasMoreTokens()) {
			System.out.print(tokenizer.nextToken() + ".");
		}		
	}
}